src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine.cs

```diff
@@ -159,6 +159,10 @@ private async Task ReportGroupsAsync(ImmutableArray<ISymbol> symbols, Cancellati
                     foreach (var groupSymbol in group.Symbols)
                         _symbolToGroup.TryAdd(groupSymbol, group);
 
+                    // Since "symbol" was in group.Symbols, and we just added links from all of group.Symbols to that group, then "symbol" 
+                    // better now be in _symbolToGroup.
+                    Contract.ThrowIfFalse(_symbolToGroup.ContainsKey(symbol));
```

Comment:
In what scenario would the symbol not be in _symbolToGroup?

Reply:
no clue.  it shouldn't happen.  but jason has hit code *later* on that crashes with that precise problem.  so i'm trying to see if we're somehow setting up this dictionary incorrectly here.